src/java/org/apache/cassandra/cql3/constraints/AbstractFunctionConstraint.java

```diff
@@ -47,7 +47,7 @@ public String term()
         return term;
     }
 
-    public abstract Set<Operator> getSupportedOperators();
+    public abstract List<Operator> getSupportedOperators();
```

Comment:
I reworked it to List because Set does not always preservers order and it might show in error messages etc and it is hard to test the exact output then.